src/Module/Acctlink.php

```diff
@@ -24,13 +24,14 @@
 use Friendica\BaseModule;
 use Friendica\Core\System;
 use Friendica\Model\Contact;
+use Friendica\Network\HTTPException\NotFoundException;
 
 /**
  * Redirects to another URL based on the parameter 'addr'
  */
 class Acctlink extends BaseModule
 {
-	public function content(): string
+	public function rawContent()
 	{
 		$addr = trim($_GET['addr'] ?? '');
```

Comment:
Whole rewrite of the method:
```suggestion
	public function rawContent()
	{
		$addr = trim($_GET['addr'] ?? '');
		if (!$addr) {
			throw new NotFoundException('Parameter "addr" is missing or empty');
		}
		
		$contact = Contact::getByURL($addr, null, ['url']) ?? '';
		if (!$contact) {
			throw new NotFoundException('Contact not found');
		}
		
		System::externalRedirect($contact['url']);
	}
```